nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -67,27 +116,39 @@ in
         # Allow systemd dependencies on vboxguest.
         SUBSYSTEM=="misc", KERNEL=="vboxguest", TAG+="systemd"
       '';
-  } (mkIf cfg.x11 {
-    services.xserver.videoDrivers = [ "vmware" "virtualbox" "modesetting" ];
-
-    services.xserver.config =
-      ''
-        Section "InputDevice"
-          Identifier "VBoxMouse"
-          Driver "vboxmouse"
-        EndSection
-      '';
-
-    services.xserver.serverLayoutSection =
-      ''
-        InputDevice "VBoxMouse"
-      '';
-
-    services.xserver.displayManager.sessionCommands =
-      ''
-        PATH=${makeBinPath [ pkgs.gnugrep pkgs.which pkgs.xorg.xorgserver.out ]}:$PATH \
-          ${kernel.virtualboxGuestAdditions}/bin/VBoxClient-all
-      '';
-  })]);
+  }
+    (
+      mkIf cfg.clipboard {
+        systemd.user.services.virtualboxClientClipboard = mkVirtualBoxUserService "--clipboard";
```

Comment:
Do those actually all need to be separate processes? Or can we shove all the args into one?

Reply:
As said below, we also want `--draganddrop`.

Reply:
`--draganddrop` is still missing?

Reply:
It was removed again as requested by @blitz as it did not work as expected